File: Search Query Key/search_query_key.js
const axios = require('axios');

const serviceName = 'your-azure-search-service';
const indexName = 'your-index-name';
const apiVersion = '2020-06-30';
const queryKey = 'your_azure_search_query_key_here';
const url = `https://${serviceName}.search.windows.net/indexes/${indexName}/docs?api-version=${apiVersion}&search=*`;

const headers = {
    'api-key': queryKey,
    'Content-Type': 'application/json'
};

axios.get(url, { headers })
    .then(response => {
        console.log(response.data);
    })
    .catch(error => {
        console.error(error);
    });

File: Search Query Key/search_query_key.py
import requests

service_name = "your-azure-search-service"
index_name = "your-index-name"
api_version = "2020-06-30"
query_key = "your_azure_search_query_key_here"
url = f"https://{service_name}.search.windows.net/indexes/{index_name}/docs?api-version={api_version}&search=*"

headers = {"api-key": query_key, "Content-Type": "application/json"}

response = requests.get(url, headers=headers)
print(response.json())
